docker/cypress-E2E-tests-postgres.yml

```
@@ -0,0 +1,56 @@
+# ----------------------------------------------------------------------------------------------------------------------
+# Cypress Setup Postgres
+# ----------------------------------------------------------------------------------------------------------------------
+
+services:
+    postgres:
```

```suggestion
    postgresql:
```

Would use the same service names as in the base services to cause less confusion when debugging.